src/constants/messages.js

```diff
@@ -0,0 +1,31 @@
+// 기본 출력 메시지
+export const MESSAGES = {
+  START: "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)",
+  ANSWER_COUNT: "시도할 횟수는 몇 회인가요?",
+  RESULT: "실행 결과",
+  WINNER: "최종 우승자 : ",
+  PROGRESS_BAR: "-",
+};
+
+// 규칙
+export const RULES = {
+  CAR_MAX_LENGTH: 5,
+  RANDOM_MIN_NUMBER: 0,
```

Comment:
```suggestion
  RANDOM_MIN_NUMBER: 0,
```
저는 문제를 잘못 읽고 1로 했었는데 소은님꺼 보고 알았네요 감사합니다!